resources/views/docs/info.blade.php

```diff
@@ -304,6 +304,9 @@
 
 ## Breaking Changes
 
+### 2021-06-14
+- Removed `description` from [Group](#group) and [UserGroup](#usergroup). It has been moved to an optional attribute on [Group](#group).
```

Comment:
More like removed from UserGroup and changed to optional attribute with different structure on Group.